app/views/support_interface/math_gcse_forms/_math_gcse_form.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= f.govuk_error_summary %>
+
+<h1 class="govuk-heading-l">Edit GCSE</h1>
+
+<%= f.govuk_radio_buttons_fieldset :qualification_type,
+  legend: { text: t('gcse_edit_type.page_titles.maths'), size: 'l' } do %>
+    <%= f.govuk_radio_button :qualification_type, :gcse, label: { text: t('application_form.gcse.qualification_types.gcse') } %>
+    <%= f.govuk_radio_button :qualification_type, :gce_o_level, label: { text: t('application_form.gcse.qualification_types.gce_o_level') } %>
+    <%= f.govuk_radio_button :qualification_type, :scottish_national_5, label: { text: t('application_form.gcse.qualification_types.scottish_national_5') } %>
+    <%= f.govuk_radio_button :qualification_type, :other_uk, label: { text: t('application_form.gcse.qualification_types.other_uk') } do %>
+      <% f.govuk_text_field :other_uk_qualification_type, label: { text: t('application_form.gcse.other_uk.label'), size: 's' } %>
+    <% end %>
+
+    <%= f.govuk_radio_divider %>
+
+    <%= f.govuk_radio_button :qualification_type, :non_uk, label: { text: t('application_form.gcse.qualification_types.non_uk') } do %>
+      <%= f.govuk_text_field :non_uk_qualification_type, label: { text: t('application_form.gcse.non_uk.label'), size: 's' }, hint: { text: t('application_form.gcse.non_uk.hint_text') } %>
+      <%= f.govuk_collection_select :institution_country, select_country_options, :id, :name, label: { text: t('gcse_edit_institution_country.page_title', subject: f.object.subject.capitalize) } %>
```

Comment:
Also, should `In which country or territory did you study for your Maths qualification?` be bold like name and enic?

<img width="531" alt="image" src="https://github.com/DFE-Digital/apply-for-teacher-training/assets/50492247/51ac59b7-5e76-4511-b515-1af24c6a2b71">